geomstats/vectorization.py

```diff
@@ -106,6 +111,36 @@ def repeat_point(point, n_reps=2, expand=False):
     return gs.repeat(gs.expand_dims(point, 0), n_reps, axis=0)
 
 
+def _is_not_none(value):
+    return value is not None
+
+
+def repeat_out(space, out, *points, out_shape=()):
+    """Repeat out shape after finding batch shape.
+
+    Parameters
+    ----------
+    space : Manifold
+        Space to which point belongs.
+    out : array-like
+        Output to be repeated
+    point : array-like
```

Comment:
points?

Reply:
good catch. in fact the parameter should be `point` for consistency with other functions, so I'll change it instead.